### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by alt-jero**
Allow usage with Socialite 3.0

### CODE REVIEW

**Comment by alt-jero** [comment]
Apparently changing a version number causes a bunch of tests to fail... I don't know how this Travis CI works, but the change works fine locally with Laravel 5.1 and socialite 3.0@dev.

**Comment by DraperStudio** [comment]
Wrong Laravel Socialite version (Travis errors) and StyleCI issues, fix them and I will take another look.

Also you should allow 2.0 and 3.0@dev because otherwise people who use 2.0 won't be able to use it.

**Comment by AndyWendt** [comment]
This may fix issue #17 but need to double check.

This is bombing out because of composer: `Your requirements could not be resolved to an installable set of packages.`

**Comment by DraperStudio** [comment]
Fine with me if we have a 2.0 branch. When the issues here get fixed I will take another look and will merge it.

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "require": {
     "php": ">=5.4.0",
     "laravel/framework": "~5.0",
-    "laravel/socialite": "~2.0"
+    "laravel/socialite": "~3.0"
```
Comment (DraperStudio) [inline composer.json]: Version should be **3.0.*@dev** because Socialite 3.0 isn't tagged as stable as of now.

**Comment by AndyWendt** [comment]
Socialite 3.0 is still in dev and to require it you would have to pull in using dev-master.  Our minimum stability is `stable` for dependencies so it doesn't allow this.

**Comment by DraperStudio** [comment]
Guess we will have to require Guzzle 6 only then or I maybe later ask Taylor when he plans to tag it as stable since it has been tagged as dev now for quite a few months without any changes.

**Comment by AndyWendt** [comment]
According to @DraperStudio conversation with Taylor Otwell, Socialite 3.0 is not even a thing yet even though it has it on packagist.

**Pull request closed without merging**